seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/sink/DruidOutputFormat.java

```diff
@@ -56,7 +56,7 @@
 import java.util.Arrays;
 import java.util.Collections;
 
-public class DruidOutputFormat extends RichOutputFormat<Row> {
+public class DruidOutputFormat<T> extends RichOutputFormat<T> {
```

Comment:
From the writeRecord, we can see it just support Row, why add the <T> for DruidOutputFormat? Could you revert this change? If no, please add some details, thanks. 

I think this change will delay the Error until writeRecord. And it just log.ERROR, flink job will run normally, right?